### DISCUSSION TITLE
Expose the GitHub PR title as an env var in Actions

### DISCUSSION THREAD

**Discussion by tas50** [community/community#30470] [Actions] [answered]
Hi there,

I work on a security product that currently integrates with GitHub to perform repo scanning via a GitHub Action we publish. We gather information about the run from the various GITHUB_* env vars and pass that data onto our platform where we expose the GitHub data + our scan results in our UI. We noticed that GitHub does not expose the PR title in the current list of env vars. This data would be incredibly useful for us to give users a better experience on our platform. Any chance we could get a new ENV var for that one?

Thanks,
Tim

**Comment by juliandunn** [comment, marked as answer]
We don't (and probably won't) create env vars for each and every item that comes across a webhook payload, but you can access the title of a PR out of the webhook context e.g. in an action's JS,

```
const title = github.context.payload.pull_request.title;
```

Would that do the trick?
